client/hwlib/src/collectors/os_info.rs

```diff
@@ -134,17 +107,10 @@ mod tests {
 
     #[test]
     fn test_os_try_new() {
-        let mock_calls = vec![
-            ((LSB_RELEASE, vec!["-c"]), Ok("Codename: focal\n")),
-            ((LSB_RELEASE, vec!["-i"]), Ok("Distributor ID: Ubuntu\n")),
-            (
-                (LSB_RELEASE, vec!["-r"]),
-                Ok("No LSB modules are available.\nRelease: 20.04\n"),
-            ),
-            ((LSMOD, vec![]), Ok("Module Size Used\nsnd 61440 1\n")),
-        ];
+        let mock_calls = vec![((LSMOD, vec![]), Ok("Module Size Used\nsnd 61440 1\n"))];
```

Comment:
Feel free to ignore this, but for consistency with creating other empty containers, I think it would be nicer to use `Vec::new()`. This has two advantages:
1. When creating other empty containers, e.g. `HashMap`, `HashMap::new()` exists but `hashmap![]` does not
2. The docs provide us with a guarantee that `Vec::new()` does not allocate. The `vec![]` macro does not provide this guarantee

As these are tests, reducing allocations doesn't matter too much so going for the more concise option is also acceptable, but I see elsewhere in this codebase `Vec::new()` is already used